manim/mobject/svg/text_mobject.py

```
@@ -575,3 +578,331 @@ def change_alignment_for_a_line(self, alignment, line_no):
+            return file_name
+        surface = cairocffi.SVGSurface(file_name, 600, 400)
+        context = cairocffi.Context(surface)
+        context.move_to(START_X, START_Y)
+        settings = self.text2settings()
+        offset_x = 0
+        last_line_num = 0
+        for setting in settings:
+            family = setting.font
+            style = self.str2style(setting.slant)
+            weight = self.str2weight(setting.weight)
+            text = self.text[setting.start : setting.end].replace("\n", " ")
+            layout = pangocairocffi.create_layout(context)
+            layout.set_width(pangocffi.units_from_double(600))
+            fontdesc = pangocffi.FontDescription()
+            fontdesc.set_size(pangocffi.units_from_double(size))
+            if family:
+                fontdesc.set_family(family)
+            fontdesc.set_style(style)
+            fontdesc.set_weight(weight)
+            layout.set_font_description(fontdesc)
+            if setting.line_num != last_line_num:
+                offset_x = 0
+                last_line_num = setting.line_num
+            context.move_to(
+                START_X + offset_x, START_Y + line_spacing * setting.line_num
+            )
+            layout.set_text(text)
+            logger.debug(f"Setting Text {text}")
+            pangocairocffi.show_layout(context, layout)
+            offset_x += layout.get_extents()[0].x
+        surface.finish()
+        return file_name
+
+
+class Text(CairoText):
+    """Display (non-LaTeX) text.
+
+    Text objects behave like a :class:`.VGroup`-like iterable of all characters
+    in the given text. In particular, slicing is possible.
+
+    .. WARNING::
+
+        Using a :class:`.Transform` on text with leading whitespace can look
+        `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
+        :meth:`remove_invisible_chars` to resolve this issue.
+
+    """
+
+    def __init__(self, text, **config):
+        logger.warning(
```

I think we should raise DeprecationWarning instead of just logging